lib/src/main/java/com/group3/racingbot/standings/Standings.java

```diff
@@ -0,0 +1,317 @@
+/**
+ * 
+ */
+package com.group3.racingbot.standings;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.function.Predicate;
+
+import org.bson.codecs.pojo.annotations.BsonCreator;
+import org.bson.codecs.pojo.annotations.BsonProperty;
+
+import com.group3.racingbot.driverstate.FinishedRace;
+import com.group3.racingbot.driverstate.Racing;
+import com.group3.racingbot.inventory.Iterator;
+//import com.group3.racingbot.inventory.Iterator;
+import com.group3.racingbot.inventory.NotFoundException;
+import com.group3.racingbot.sorting.DriverStandingsHeapSort;
+import com.group3.racingbot.sorting.DriverStandingsQuickSort;
+import com.group3.racingbot.sorting.SortStandings;
+
+/**
+ * Keeps track of the pole positions of all drivers within a race event.
+ * @author Nick Sabia
+ */
+public class Standings {
+	private List<DriverStanding> standings;
+	private String raceEventId;
+	
+	/**
+	 * Construct the Standings. Holds all drivers which are participating in the race event whose id is supplied.
+	 * @param raceEventId the race event id of the race event which the standings are for
+	 */
+	@BsonCreator
+	public Standings(@BsonProperty("raceEventId") String raceEventId) {
+		this.standings = new ArrayList<DriverStanding>();
+		this.raceEventId = raceEventId;
+	}
+	
+	/**
+	 * Sorts the standings based on driver's distance traveled to find out their positions.
+	 */
+	public void sortStandings() {
+		// Create four lists.
+		List<DriverStanding> finished = new ArrayList<DriverStanding>(); // This list holds all drivers which have completed the race.
+		List<DriverStanding> racing = new ArrayList<DriverStanding>(); // This list holds all drivers who are currently racing.
+		List<DriverStanding> dnf = new ArrayList<DriverStanding>(); // This list holds all drivers who couldn't complete the race.
+		List<DriverStanding> result = new ArrayList<DriverStanding>(); // This list holds all drivers in the order which they stand within the race event.
+		// Defines the sorting algorithm to use
+		SortStandings sortAlgorithm = new DriverStandingsQuickSort();
```

Comment:
This would be better example of Strategy pattern if sort algorithm was held or passed in rather than hard coded here (i.e. some context (the strategy to be used) is held or used.